Model/Config/Source/Status/AdyenStates.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+
+namespace Adyen\Payment\Model\Config\Source\Status;
+
+
+class AdyenStates
```

Comment:
Can you rename it to the singular `AdyenState` and add a new line at the end of the file?